subiquity/server/server.py

```diff
@@ -123,6 +123,10 @@ async def client_variant_POST(self, variant: str) -> None:
             self.app.controllers.Source.configured()
         else:
             self.app.base_model.set_source_variant(variant)
+        self.app.variant = variant
```

Comment:
It would be more consistent for this to call self.app.set_source_variant would it not?

Reply:
done for the fixes